src/renderer/services/valuation.js

```diff
@@ -161,4 +161,134 @@ export default {
     });
   },
 
+  getBTCBalanceByAddress(walletClient) {
```

Comment:
> It would be nice to add a comment for this about the format of the response returned from this this. If this is documented elsewhere since this is from bitpay APIs at least a link to them could help.

Okay sure 
 
> It also doesn't seem like thee methods belong in valuation.js, there should probably be a new file for these external APIs to work with bitcoin.

New file bwclient.js